src/components/CtaSection.tsx

```diff
@@ -158,7 +170,7 @@ export const CtaSection: FunctionComponent<CtaSection> = ({
                 })}
             >
                 {cta1 && (
-                    <div className={classNames({ 'tw-mt-sm': !slimWidth })}>
+                    <div className={classNames({ 'tw-mt-sm': !slimWidth, 'tw-mr-sm': centerContent })}>
```

Comment:
Let's add that margin after the `xs` breakpoint when the buttons are side by side since we don't want this when they're stacked